common/tier4_planning_rviz_plugin/include/path/display.hpp

```diff
@@ -60,17 +62,27 @@ private Q_SLOTS:
     const double vel_max, const double cmd_vel);
   static std::unique_ptr<Ogre::ColourValue> gradation(
     const QColor & color_min, const QColor & color_max, const double ratio);
+
   Ogre::ManualObject * path_manual_object_{nullptr};
   Ogre::ManualObject * velocity_manual_object_{nullptr};
+  rviz_rendering::BillboardLine * left_bound_line_{nullptr};
+  rviz_rendering::BillboardLine * right_bound_line_{nullptr};
```

Comment:
Could you please create it in TRIANGLE instead of BILLBOARD? The billboard would rotate with the viewpoint.

Reply:
Thank you for your comment, but if I use TRIANGLE I have to compute the angle and orthogonal vectors for changing the width of the line

Reply:
If it is possible, I would like to use BILLBOARD. (I intentionally use Billboard insteadd of TRIANGLE)

Reply:
I do not want to use billboard if it is possible.
We also avoid the use of billboard for trajectory, path, lane(linestring) and etc., and use triangles.

Reply:
![image (2)](https://user-images.githubusercontent.com/43805014/218326476-2f1990fb-a9d5-40b3-b5b4-5fee60fb5fa6.png)

This is the result when I use triangle

Reply:
@yukkysaito I changed it to use triangle